databuilder/databuilder/extractor/es_base_extractor.py

```diff
@@ -30,6 +45,18 @@ def init(self, conf: ConfigTree) -> None:
 
         self.es = self.conf.get(ElasticsearchBaseExtractor.ELASTICSEARCH_CLIENT_CONFIG_KEY)
 
+    def _get_es_version(self) -> str:
+        version = ''
+
+        try:
+            info = self.es.info()
+            if info:
+                version = info.get('version').get('number')
+        except Exception:
+            pass
+
+        return version
```

Comment:
This function currently allows scenario where value = '' is returned. Maybe we should raise an error if that's the case instead of allowing it to happen?

Reply:
I agree that it is not good to allow `value = ''`. The scenario where we are unable to retrieve Elasticsearch version would imply a deep enough issue that raising an error would be the best course of action. I updated the code to remove the try/catch because the underlying `info()` call will raise its own error if it encounters a problem.